ui/components/home/Callout1.tsx

```diff
@@ -27,10 +27,9 @@ export default function Callout1() {
               id="paragraph"
               className="pt-2 pb-5 text-black text-lg w-[100%] max-w-[500px]"
             >
-              We are an open source space community where everything is
+              We are an open, global space community where everything is
```

Comment:
since this is specifically about the community on Discord, I thought this was more accurately representative of that, but can change back if need be. But the discord community isn't necessarily "open source"